client/state/login/actions.js

```diff
@@ -23,10 +23,10 @@ import {
 } from 'state/action-types';
 
 const loginErrorMessages = {
-	empty_password: translate( 'The password field is empty.' ),
-	empty_username: translate( 'The username field is empty.' ),
-	incorrect_password: translate( 'Invalid password' ),
-	invalid_two_step_code: translate( 'Invalid verification code.' ),
+	empty_password: translate( 'Enter your password.' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **7** times:
`translate( 'Please enter your password' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/login-errors)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).